bootstrap/lib/cache.js

```diff
@@ -26,6 +33,8 @@ const envOptions = () => ({
     intervalMin: process.env.REQUEST_COALESCING_INTERVAL_MIN || 100,
     intervalMax: process.env.REQUEST_COALESCING_INTERVAL_MAX || 1000,
     intervalCoefficient: process.env.REQUEST_COALESCING_INTERVAL_COEFFICIENT || 2,
+    // Add entropy to absorb bursts
+    entropyMax: process.env.REQUEST_COALESCING_ENTROPY_MAX || 0,
```

Comment:
I would definitely add example configurations for the new values added

Reply:
I've added some docs in [c4a3db4](https://github.com/smartcontractkit/external-adapters-js/pull/93/commits/c4a3db4c2ff3eae5c1533648ee4fd810648a1088). Would that be enough or did you have something else in mind?

Reply:
No these look great, thanks!